docs/docs/contributions/releases/release-process.mdx

```diff
@@ -158,6 +158,26 @@ Once the workflow finishes, look through any failures and determine if there's a
 
 Alternatively, after starting the workflow, post the link to the in-progress run in `#development` in Slack, so that someone can come back to it when it does finish.
 
+## Step 5: Publish a schema in JSON Schema Store
+
+Pants configuration file schema is published at https://www.schemastore.org/. For every release, a new schema needs to generated
```

Comment:
Also, a few questions:

It looks like we only do this for stable `.0` releases, not each dev/alpha/RC release, or even `.1`/`.2`/... patch releases. Is that correct?

```suggestion
Pants configuration file schema is published at https://www.schemastore.org/. For every stable `2.x.0` release, a new schema needs to generated
```

Reply:
Yes, that's correct! Updated.